utilities/common.py

```diff
@@ -105,14 +105,17 @@ def common_parser(for_reco_highPU=False):
     parser.add_argument("-e", "--era", type=str, choices=["2016PreVFP","2016PostVFP"], help="Data set to process", default="2016PostVFP")
     if for_reco_highPU:
         # additional arguments specific for histmaker of reconstructed objects at high pileup (mw, mz_wlike, and mz_dilepton)
-        parser.add_argument("--muonCorr", type=str, default="massfitData_lblidealMC", 
-            choices=["none", "trackfit_only", "trackfit_only_mctruth", "lbl", "massfit", "massfit_lbl", "massfitData_mctruth"], 
-            help="Type of correction to apply to the muons")
+        parser.add_argument("--muonCorrMC", type=str, default="idealMC_lbltruth", 
+            choices=["none", "trackfit_only", "trackfit_only_idealMC", "lbl", "idealMC_lbltruth", "idealMC_massfit", "idealMC_lbltruth_massfit"], 
+            help="Type of correction to apply to the muons in simulation")
+        parser.add_argument("--muonCorrData", type=str, default="lbl_massfit", 
```

Comment:
Should we switch to a more consistent format with dashes (e.g., muon-corr-mc)

Reply:
We have no consistency at all in the use of the arguments, there are all types: camel case, dash, or underline
If you want I can clean it up and make it consistently "dashed" like